src/manifolds/HyperbolicHyperboloid.jl

```diff
@@ -1,3 +1,25 @@
+@doc raw"""
+    change_representer(M::Hyperbolic{n}, ::EuclideanMetric, p, X)
+
+Change a Eucliden gradient from the embedding that was already projected onto the tangent space at `p`.
+We only have to correct for the metric, which means that the sign of the last entry changes, since
+for the result ``Y``  we are looking for a tangent vector such that
+
+```math
+    g_p(Y,Z) = -y_{n+1}z_{n+1} + \sum_{i=1}^n y_iz_i = \sum_{i=1}^{n+1} z_ix_i
+```
+
+holds, which directly yields ``y_i=x_i`` for ``i=1,\ldots,n`` and ``y_{n+1}=-x_{n+1}``.
+"""
+change_representer(::Hyperbolic, ::EuclideanMetric, ::Any, ::Any)
```

Comment:
Could you add a reference with Riesz representation theorem stated for non-positive-definite inner products? So far I've only seen it for positive-definite inner products.

Reply:
Are you sure we need that here?
The `EuclideanMetric` is a metric (i.e. the matrix representation is SPD), but the `MinkowskiMetric` is also a metric (i.e. the inner product positive definite) – _when_ we restrict Minkowski to a tangent space of the Hyperboloid model. That is actually a neat small exercise to do, but that way both Euclidean (from the embedding) and Minkowski are metric (positive definite inner products) on TpM and this can be done here.

Reply:
Ah, right, in this case we don't need that.